client/components/domains/register-domain-step/index.jsx

```diff
@@ -648,7 +648,7 @@ class RegisterDomainStep extends Component {
 					title=""
 					className="register-domain-step__placeholder"
 					illustration={ Illustration }
-					illustrationWidth={ 180 }
+					illustrationWidth={ 280 }
```

Comment:
Setting the width to 280 here makes the height of the new illustration about the same as the former illustration.